### PR TITLE
fix(tag): update reduce method in story

### PR DESCRIPTION
**Pull request by jendowns**
Small thing --

I wanted to suggest updating the first argument in the `reduce` method here from `o` to `accumulator`. I think that in a case where there's nothing relevant to name the accumulator, it would be best to just use a default / recognizable word to help with readability. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#Syntax

#### Changelog

**Changed**

- change first argument of `reduce` method from `o` to `accumulator`

### CODE REVIEW

**Comment by asudoh** [comment]
Heads-up @jendowns - The repo has been merged to the vanilla repo. Good to have your future contributions there - Thanks!

File: src/components/Tag/Tag-story.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ import Tag, { types as typesList } from '../Tag';
 import TagSkeleton from '../Tag/Tag.Skeleton';
 
 const types = typesList.reduce(
-  (o, type) => ({
-    ...o,
+  (accumulator, type) => ({
```
Comment (joshblack) [inline src/components/Tag/Tag-story.js]: Super super, uber, mega nitpicky, but we tend to abbreviate as `acc`:

```suggestion
  (acc, type) => ({
```

Comment (jendowns) [reply src/components/Tag/Tag-story.js]: No problem @joshblack! I just committed your change 👍

**Comment by jendowns** [comment]
Hey it looks like the build for my latest commit was cancelled as a redundant build. Is anyone able to restart it for me? 😅

**Comment by asudoh** [comment]
Superseded by carbon-design-system/carbon#2683 - @jendowns Our apologies for the hustle!

**Pull request closed without merging**